etcdmain/etcd.go

```diff
@@ -127,7 +128,7 @@ func startEtcdOrProxyV2() {
 
 	var stopped <-chan struct{}
 	var errc <-chan error
-
+	var lerrc <-chan error
```

Comment:
What does `lerrc` stand for?

Reply:
I think thats a mistake, its supposed to be listener error channel, fixing that